### ISSUE TITLE
Copy_Move Forgery detection using CNN

### ISSUE THREAD

**Issue by aliretha** [vlfeat/matconvnet#673]
Hi,

I am trying to train the CNN to detect forgery on images, I have created ~30,000 colored images  ( 15,000 original images without forgery and 15,000 has forgery ). Then I have used 22,500 for training and 7,500 for validating.  
The result that I have got as the figure below.
![result](https://cloud.githubusercontent.com/assets/6691308/18125911/ae991b84-6f70-11e6-923b-e1dd92601be5.png)

Consider the network that I used is
`opts.networkType = 'simplenn' ;
opts = vl_argparse(opts, varargin) ;
 lr = [0.0001 0.0002] ;
net.layers = {} ;
% Block 1
net.layers{end+1} = struct('type', 'conv', ...
                           'weights', {{0.01*randn(8,8,3,32, 'single'), zeros(1, 32, 'single')}}, ...
                           'learningRate', lr, ...
                           'stride', 1, ...
                           'pad', 2) ;
net.layers{end+1} = struct('type', 'pool', ...
                           'method', 'max', ...
                           'pool', [3 3], ...
                           'stride', 2, ...
                           'pad', [0 1 0 1]) ;
net.layers{end+1} = struct('type', 'relu') ;
% Block 2
net.layers{end+1} = struct('type', 'conv', ...
                           'weights', {{0.05*randn(8,8,32,32, 'single'), zeros(1,32,'single')}}, ...
                           'learningRate', lr, ...
                           'stride', 1, ...
                           'pad', 2) ;
net.layers{end+1} = struct('type', 'relu') ;
net.layers{end+1} = struct('type', 'pool', ...
                           'method', 'avg', ...
                           'pool', [3 3], ...
                           'stride', 2, ...
                           'pad', [0 1 0 1]) ; % Emulate caffe
% Block 3
net.layers{end+1} = struct('type', 'conv', ...
                           'weights', {{0.05*randn(8,8,32,32, 'single'), zeros(1,32,'single')}}, ...
                           'learningRate', lr, ...
                           'stride', 1, ...
                           'pad', 2) ;
net.layers{end+1} = struct('type', 'relu') ;
net.layers{end+1} = struct('type', 'pool', ...
                           'method', 'avg', ...
                           'pool', [3 3], ...
                           'stride', 2, ...
                           'pad', [0 1 0 1]) ; % Emulate caffe
% Block 4
net.layers{end+1} = struct('type', 'conv', ...
                           'weights', {{0.05*randn(4,4,32,32, 'single'), zeros(1,32,'single')}}, ...
                           'learningRate', lr, ...
                           'stride', 1, ...
                           'pad', 0) ;
net.layers{end+1} = struct('type', 'relu') ;
% Block 5
net.layers{end+1} = struct('type', 'conv', ...
                           'weights', {{0.05*randn(1,1,32,10, 'single'), zeros(1,10,'single')}}, ...
                           'learningRate', [0.0001 0.0002], ...
                           'stride', 1, ...
                           'pad', 0) ;
% Loss layer
net.layers{end+1} = struct('type', 'softmaxloss') ;

% Meta parameters
net.meta.inputSize = [128 128 3] ;
net.meta.trainOpts.learningRate = [0.05*ones(1,30) 0.005*ones(1,10) 0.0005*ones(1,5)] ;
net.meta.trainOpts.weightDecay = 0.0001 ;
net.meta.trainOpts.batchSize = 1 ;
net.meta.trainOpts.numEpochs = numel(net.meta.trainOpts.learningRate) ; 
`
Kindly, could you help me to find where is the problem ?
